### PR TITLE
legoland/ragequit

### PR DESCRIPTION
**Pull request by pixelranchllc**
RageQuit Form Refactor
https://app.clickup.com/t/y3jwqc

Refactor of form, fields, customValidators, and profile menu logic to replace forms/rageQuit.jsx.

### CODE REVIEW

File: src/utils/validation.js [hunk 1]
```diff
@@ -122,4 +122,32 @@ export const customValidations = {
     }
     return false;
   },
+  rageQuitMinimum({ values }) {
+    if (
+      (values.shares === '' || values.shares === 0 || values.shares === '0') &&
```
Comment (skuhlmann) [inline src/utils/validation.js]: is en empty string falsey?

wonder if we could just do something like:

```
if (!values.shares && !values.loot)
```

**Review by skuhlmann — CHANGES_REQUESTED** [review]

**Comment by pixelranchllc** [comment]
Code change based on comment above.

**Comment by pixelranchllc** [comment]
Fix to error on empty shares or loot field.
Fix to spelling error in validator.
Form modal closes properly on submit.

File: src/utils/validation.js (hunk 1 above)
Comment (pixelranchllc) [reply src/utils/validation.js]: Changed

**Pull request closed without merging**